contributing-guides/style-guide.md

```diff
@@ -481,6 +478,19 @@ When documenting optional placeholders like paths or file extensions, it is sugg
 
 - Use `{{path/to/source.ext}}` instead of `{{path/to/source.tar[.gz|.bz2|.xz]}}`.
 
+### Keypress syntax
+
+To mark keypresses for TUI or UI programs, use angle brackets `<` and `>`.
+
+- Single characters example: `<a>`.
+- Special keys are to be written with [`PascalCase`](https://en.wikipedia.org/wiki/Camel_case): `<Ctrl>`, `<Super>`, `<Alt>`, `<Shift>`, `<Cmd>`, `<Option>`, `<Windows>`, `<Enter>`, `<Home>`, `<Space>`, `<Esc>`, `<ArrowUp>`, `<ArrowLeft>`, `<PageUp>`, `<F5>`, `<F12>`, ...
+- Special keys can be translated if they have culturally relevant translations.
+- When a program takes in uppercase character literals mark them as `<A>` instead of marking it with shift. Otherwise always mark characters in lowercase.
```

Comment:
Any opinions on this? You always see terminal keypresses marked with uppercase in documentation, but during my brainstorming this made more logical sense.
The lowercase letters do feel less impactful, that's for sure.

Reply:
IG this is fine, but it might confuse a few new users using tldr but for people used to CLI commands this won't be an issue, also if we added "Shift" and other keys to the example it will increase it's overall size thus making the rendering large too.